src/Symfony/Component/Cache/Tests/Adapter/FilesystemAdapterTest.php

```diff
@@ -49,4 +49,62 @@ public static function rmdir($dir)
         }
         rmdir($dir);
     }
+
+    public function testPrune()
```

Comment:
The four test cases don't share a common parent class within this component's scope. Should I create a trait that can be used by `AdapterTestCase` and `CacheTestCase`? Or perhaps a new `AbstractCacheTestCase` that the aforementioned classes can inherit from? Or should the logic be duplicated between the `Adapter` and `Simple` top-level test classes?